cni/network/network.go

```diff
@@ -666,7 +676,7 @@ func (plugin *NetPlugin) createNetworkInternal(
 
 // construct network info with ipv4/ipv6 subnets
 func addSubnetToNetworkInfo(ipamAddResult IPAMAddResult, nwInfo *network.NetworkInfo) error {
-	for _, ipConfig := range ipamAddResult.defaultInterfaceInfo.IPConfigs {
+	for _, ipConfig := range ipamAddResult.secondaryInterfacesInfo[0].IPConfigs {
```

Comment:
add branching or call different function on L678 (new function for secondary nics)